src/docs/self-hosted/geolocation.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Self-Hosted Geolocation'
+---
+
+Sentry can use [MaxMind's free GeoLite2 database](https://dev.maxmind.com/geoip/geoip2/geolite2/) to geolocate IP addresses, providing additional context for error events where the end-user's IP address is known, as well as for the session history of users logging into your Sentry installation. Our on-premise offering bundles MaxMind's [`geoipupdate`](https://hub.docker.com/r/maxmindinc/geoipupdate) tool for initially downloading and periodically updating this dataset. To enable this functionality, you should sign up for [a free MaxMind account](https://www.maxmind.com/en/geolite2/signup), and then tell Sentry about your credentials via a [`docker-compose.override.yml`](https://docs.docker.com/compose/extends/#understanding-multiple-compose-files) file, like so:
```

Comment:
Nitz:
```suggestion
Sentry can use [MaxMind's free GeoLite2 database](https://dev.maxmind.com/geoip/geoip2/geolite2/) to geolocate IP addresses, providing additional context for error events where the end-user's IP address is known, as well as for the session history of users logging into your Sentry installation. Our self-hosted offering bundles MaxMind's [`geoipupdate`](https://hub.docker.com/r/maxmindinc/geoipupdate) tool for initially downloading and periodically updating this dataset. To enable this functionality, you should [sign up for a free MaxMind account](https://www.maxmind.com/en/geolite2/signup), and then tell Sentry about your credentials via a [`docker-compose.override.yml`](https://docs.docker.com/compose/extends/#understanding-multiple-compose-files) file, like so:
```

Also, re `docker-compose.override.yml`: let's take it slow and encourage these env variables for now instead?